homeassistant/components/rachio/switch.py

```diff
@@ -318,20 +313,25 @@ def name(self) -> str:
     @property
     def unique_id(self) -> str:
         """Return a unique id by combining controller id and schedule."""
-        return f"{self._controller.controller_id}-schedule-{self.schedule_id}"
+        return f"{self._controller.controller_id}-schedule-{self._schedule_id}"
 
     @property
     def icon(self) -> str:
         """Return the icon to display."""
         return "mdi:water"
 
+    @property
```

Comment:
Does this property need to be exposed?  I don't think it used anywhere else.

Reply:
Better to just do this?
```
@property
    def device_state_attributes(self) -> dict:
        """Return the optional state attributes."""
        return {
            ATTR_SCHEDULE_SUMMARY: self._summary,
            ATTR_SCHEDULE_ENABLED: self.schedule_is_enabled,
            ATTR_SCHEDULE_DURATION: f"{round(self._duration / 60)} minutes"
        }
```